include/tscore/ink_config.h.cmake.in

```diff
@@ -165,7 +165,12 @@ const int DEFAULT_STACKSIZE = @DEFAULT_STACK_SIZE@;
 // TODO(cmcfarlen): Verify use of below in iocore/net/SSLNetVConnection (redunant)
 #cmakedefine01 HAVE_SSL_READ_EARLY_DATA
 #cmakedefine HAVE_SSL_SET_MAX_EARLY_DATA
```

Comment:
Not part of your change but I wonder whether this is a bug.

Reply:
It's not a bug, it just doesn't *look* right. But it should be changed for consistency.

Reply:
I'll make a note to do that.